rmf_demos_panel/rmf_demos_panel/gui_server.py

```diff
@@ -37,13 +37,17 @@ def download_webpack():
     # it is advice to update the version (in '.github/workflows/gh_page.yml')
     # whenever a new feature is introduced
     source_url = "https://open-rmf.github.io/rmf_demos/v1.0.0/app.bundle.js"
-
+    print("Check path ", bundle_path)
     if os.path.isfile(bundle_path):
         print("BundleJS File exists, SKIP!")
     else:
         print(f"BundleJS File does not exist, Download it from {source_url}")
         response = requests.get(source_url)
         if response.status_code == 200:
+            dir_path = os.path.dirname(bundle_path)
+            if not os.path.isdir(dir_path):
+                print(f"Dir path: {dir_path} doesnt exist, create dir")
+                os.mkdir(dir_path)
```

Comment:
Same as above

Reply:
This is related to `rmf_panel_panel`, in which to create a `dist` dir if the folder is not available. This is mentioned in the description as a minor not crucial diff. I can open another pr if this looks confusing to have it here.

Reply:
That's probably a good idea? It might be confusing in the version history to see this file change with a commit message related to crowd simulation

Reply:
Removed, (same as above): https://github.com/open-rmf/rmf_demos/pull/14/commits/89270932e657136cbf8c52f2cd135b0ea5560a85